UPDATING.md

```diff
@@ -225,8 +225,8 @@ indefinitely. This is only available on the command line.
 After how much time should an updated DAG be picked up from the filesystem.
 
 #### dag_dir_list_interval
-How often the scheduler should relist the contents of the DAG directory. If you experience that while developing your
-dags are not being picked up, have a look at this number and decrease it when necessary.
+How often the scheduler should relist the contents of the DAG directory. If while developing your
```

Comment:
`How often the scheduler should relist the contents of the DAG directory. If while developing your
+dags they are not being picked up, have a look at this number and decrease it when necessary.` --> `The frequency with which the scheduler should relist the contents of the DAG directory. If while developing
+dags, they are not being picked up, have a look at this number and decrease it when necessary.`